IntegrationTests/Services/AWSRoute53IntegrationTests/AWSRoute53Tests.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+import AWSRoute53
```

Comment:
A simple integration test for Route 53:
- creates a hosted zone,
- creates an A record in that zone,
- deletes the A record,
- deletes the hosted zone.

All operations used after creating the zone require the TrimHostedZone customization in order to succeed.